dell/xps/15-9560/default.nix

```
@@ -4,36 +4,18 @@
     ../../../common/cpu/intel/kaby-lake
     ../../../common/pc/laptop
     ./xps-common.nix
-
-    # FIXME: remove this when bumblebee works again
-    ../../../common/gpu/nvidia/disable.nix
+    ../../../common/gpu/nvidia
```

Shouldn't be that prime?

```suggestion
    ../../../common/gpu/nvidia/prime
```

---

Ah from reading the code it looks that if offload is not enabled, it doesn't use prime offload but prime sync? https://github.com/NixOS/nixpkgs/blob/bf99590fba1405cbc8c24154a2563a1f81b908d9/nixos/modules/hardware/video/nvidia.nix#L335

---

Prime sync needs to be enabled explicitly, at least the NixOS option does.

I believe if neither is enabled, the igpu is used and the nvidia gpu is just available for compute.

---

Should we import nvidia/offload such that the offload config and command is available?